packages/kubernetes-1.26/kubelet-exec-start-conf

```diff
@@ -0,0 +1,31 @@
+[Service]
+ExecStart=
+ExecStart=/usr/bin/kubelet \
+{{#unless settings.kubernetes.standalone-mode}}
+    --cloud-provider "external" \
+    --kubeconfig /etc/kubernetes/kubelet/kubeconfig \
+{{#if (eq settings.kubernetes.authentication-mode "tls")}}
+    --bootstrap-kubeconfig /etc/kubernetes/kubelet/bootstrap-kubeconfig \
+{{/if}}
+{{else}}
+    --cloud-provider "" \
```

Comment:
Does this need to be external now too since cloud providers are no longer in-tree?

Or maybe [dropped completely](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)?

> (DEPRECATED: will be removed in 1.24 or later, in favor of removing cloud provider code from Kubelet.)

Reply:
I was thinking maybe we need to keep this empty to not enable a cloud-provider for metal variants where a cloud manager might not exist?

Reply:
We can't remove it and enable to `external` since metal variants where a cloud manager might no exist need empty string `""` on.